scripts/account-faucet/src/faucet/command.ts

```diff
@@ -0,0 +1,42 @@
+import config from '../config';
+import { Command } from 'commander';
+import { Address, HexString } from '@ckb-lumos/base';
+import { GodwokenWeb3 } from '../godwoken/web3';
+import { claimFaucetForCkbAddress } from './faucet';
+import { encodeLayer2DepositAddress, privateKeyToLayer2DepositAddress, toHexString } from './address';
+
+export interface FaucetCommand {
+  privateKey?: HexString | string;
+  ckbAddress?: HexString | string;
+  ethAddress?: HexString | string;
+  rpc?: string;
+}
+
+export function setupFaucetCommand(program: Command) {
+  program
+    .command('claim')
+    .description('claim faucet for ckb account')
+    .option('-p, --private-key <privateKey>', '[HexString] ckb private key')
+    .option('-c --ckb-address <ckbAddress>', '[HexString] ckb address (optional)')
```

Comment:
Maybe we could use the this [address](https://pudge.explorer.nervos.org/address/ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqflz4emgssc6nqj4yv3nfv2sca7g9dzhscgmg28x) by default?

Reply:
> Maybe we could use the this [address](https://pudge.explorer.nervos.org/address/ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqflz4emgssc6nqj4yv3nfv2sca7g9dzhscgmg28x) as `ckbAddress` by default?

Why this specific address? 

Condition checks of this part:
- passing a single argument `privateKey`
- or passing `ckbAddress` and `ethAddress` at the same time

Reply:
> Why this specific address?

This address is the sender of [CKB Faucet](https://faucet.nervos.org/).